cfme/tests/cloud/test_providers.py

```diff
@@ -208,6 +208,17 @@ def ec2_provider_with_sts_creds(appliance):
     prov.delete()
 
 
+@pytest.fixture
+def instance_without_name(provider):
+    template_id = provider.mgmt.get_template(
+        provider.data.templates.get('small_template').name).uuid
+    instance = provider.mgmt.create_vm(template_id)
+    wait_for(lambda: instance.state == VmState.RUNNING, delay=15, timeout=900)
```

Comment:
Q: Can you use the `create_vm` fixture? https://github.com/ManageIQ/integration_tests/blob/950033f72b320943601c72878c74da3a675478ec/cfme/fixtures/vm.py#L111-L118